src/main/resources/db/changelog/changes/09-create-orders-table.yaml

```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-orders-table
+      author: anastasiia
+      changes:
+        - createTable:
+            tableName: orders
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column: 
+                  name: status
+                  type: tinyint

```

Comment:
is it the correct status?

Reply:
yes, liquibase want it to be tinyint (whatever it is))

Reply:
`tinyint` is intended to store a numeric value, but status is not such a value